packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx

```diff
@@ -296,12 +314,20 @@ export default connect(
       },
     } = state;
     const stage = stages[stageIndex];
+    assertReduxStage(stage);
+
     return {
       stageIndex,
       isEnabled: !stage?.disabled,
-      stages: stages.map((stage) => {
-        return stage.stageOperator;
-      }),
+      stages: stages.reduce<Stage[]>((accumulator, stage, idxInStore) => {
+        if (stage.type === 'stage') {
```

Comment:
Can we add test cases to this? Add wizards in the store and then validate the expected behaviour of focus mode (navigation)?

Reply:
Yes I am on it.